sdv/constraints/tabular.py

```diff
@@ -211,74 +214,96 @@ class GreaterThan(Constraint):
-        high_is_scalar(bool or None):
-            Whether or not the value for high is a scalar or a column name.
-            If ``None``, this will be determined during the ``fit`` method
-            by checking if the value provided is a column name.
-        low_is_scalar(bool or None):
-            Whether or not the value for low is a scalar or a column name.
-            If ``None``, this will be determined during the ``fit`` method
-            by checking if the value provided is a column name.
+        scalar (str):
+            Which value is a scalar. Can be ``'high'``, ``'low'`` or ``None``.
+            If ``None`` then both ``high`` and ``low`` are column names.
     """
 
-    _diff_column = None
+    _diff_columns = None
     _is_datetime = None
-    _column_to_reconstruct = None
+    _columns_to_reconstruct = None
 
     def __init__(self, low, high, strict=False, handling_strategy='transform',
-                 fit_columns_model=True, drop=None, high_is_scalar=None,
-                 low_is_scalar=None):
+                 fit_columns_model=True, drop=None, scalar=None):
+
+        if isinstance(low, list) and isinstance(high, list):
+            raise ValueError('`low` and `high` can only be multiple columns ',
+                             'if the other one is scalar.')
+        if scalar == 'high':
+            low = [low] if isinstance(low, str) else low
+            self.constraint_columns = tuple(low)
+        elif scalar == 'low':
+            high = [high] if isinstance(high, str) else high
+            self.constraint_columns = tuple(high)
+        else:
+            if isinstance(low, list):
+                raise ValueError('`low` cannot be a list when `scalar` is `None`.')
+            elif isinstance(high, list):
+                raise ValueError('`high` cannot be a list when `scalar` is `None`.')
+
+            self.constraint_columns = (low, high)
+            low = [low]
+            high = [high]
+
         self._low = low
         self._high = high
         self._strict = strict
-        self.constraint_columns = (low, high)
         self._drop = drop
-        self._high_is_scalar = high_is_scalar
-        self._low_is_scalar = low_is_scalar
+        self._scalar = scalar
+
+        if strict:
+            self.operator = np.greater
+        else:
+            self.operator = np.greater_equal
+
         super().__init__(handling_strategy=handling_strategy,
                          fit_columns_model=fit_columns_model)
 
     def _get_low_value(self, table_data):
-        if self._low_is_scalar:
+        if self._scalar == 'low':
             return self._low
-        elif self._low in table_data.columns:
-            return table_data[self._low]
+        elif any([low in table_data.columns for low in self._low]):
+            return table_data[self._low].values
 
         return None
 
     def _get_high_value(self, table_data):
-        if self._high_is_scalar:
+        if self._scalar == 'high':
```

Comment:
Same as in other places: Maybe it would be simpler to make a single, generic method instead of two?

```python3
def _get_value(self, table_data, field):
    if self._scalar == field:
        return getattr(self, f'_{field}')

    return table_data[getattr(self, f'_{field}')].values

...
low = self._get_value(table_data, 'low')
high = self._get_value(table_data, 'high')
```